react/src/components/ServiceLauncherModal.tsx

```diff
@@ -636,11 +763,64 @@ const ServiceLauncherModal: React.FC<ServiceLauncherProps> = ({
                       required
                     >
                       <Suspense fallback={<Skeleton.Input active />}>
-                        <VFolderLazyView uuid={endpoint?.model} />
+                        <VFolderLazyView
+                          uuid={endpoint?.model}
+                          mountDestination={endpoint?.model_mount_destination}
```

Comment:
If you want to display the `mountDestination` path alongside with the vFolder name, it's better to display it using one of the methods below:

As a sibling:
```
<Flex>
  <FolderLazyView/>
  <TyphoGraphy.Text>{endpoint?.model_mount_destination}</TyphoGraphy.Text>
</Flex>
```

As a child:
```
<FolderLazyView>
  <TyphoGraphy.Text>{endpoint?.model_mount_destination}</TyphoGraphy.Text>
</FolderLazyView>
```

Reply:
Thanks for the suggestion! I think it's okay to get rid of model_mount_destination since it's already applied as Input form with disabled status...